src/Icon/Icon.tsx

```diff
@@ -3650,6 +3650,20 @@ export const Icon: FC<Props> = ({
         />
       </svg>
     )}
+    {render === 'caret-down' && (
+      <svg
+        width="12"
+        height="8"
+        viewBox="0 0 12 8"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        <path
+          d="M11.7444 0.18579C11.4316 -0.0864597 10.9591 -0.0549597 10.6861 0.25704L6.00012 5.61129L1.31412 0.25629C1.04112 -0.0557097 0.567867 -0.0872097 0.255867 0.18579C-0.0561332 0.45879 -0.0868832 0.932791 0.186117 1.24404L5.43612 7.24404C5.57862 7.40754 5.78337 7.50054 6.00012 7.50054C6.21687 7.50054 6.42162 7.40754 6.56412 7.24404L11.8141 1.24404C12.0871 0.932791 12.0556 0.45879 11.7444 0.18579Z"
+          fill="#D2D2D2"
+        />
+      </svg>
+    )}
```

Comment:
you don't really need a new icon for this, we already have `caret`, you can just rotate it in the Icon component